routes/accountRouter.js

```diff
@@ -70,13 +73,50 @@ accountRouter.route('/')
 
 accountRouter.route('/:accountID')
   .options(cors.corsWithOptions, (req, res) => { res.sendStatus(200); })
-  .get(cors.cors, (req, res) => {
-    res.statusCode = 403;
-    res.end(`GET operation is not supported on /accounts/${req.params.accountID}`);
+  .get(cors.cors, (req, res, next) => {
+    AccountStats.findOne({ 'data.accountId': req.params.accountID })
+      .then((accountStats) => {
+        res.statusCode = 200;
+        res.setHeader('Content-Type', 'application/json');
+        res.json(accountStats);
+      }, (err) => next(err))
+      .catch((err) => next(err));
```

Comment:
The .then() method is replaced with async/await to handle asynchronous operations more cleanly. The try/catch block is used to handle any errors that occur during the asynchronous operation
```suggestion
  .get(cors.cors, (req, res, next) => {
      try {
        const accountStats = await AccountStats.findOne({ 'data.accountId': req.params.accountID });
        res.statusCode = 200;
        res.setHeader('Content-Type', 'application/json');
        res.json(accountStats);
      } catch (err) {
        next(err);
      }
```